scripts/lib/sphinx/SphinxToMdResult.ts

```diff
@@ -16,6 +16,7 @@ export type SphinxToMdResult = {
   markdown: string;
   meta: PythonObjectMeta;
   images: Array<{ src: string; dest: string }>;
+  isReleaseNotes?: boolean;
```

Comment:
With the `?`, do you need a tri-state: `false`, `true`, `unknown`? Or is the boolean sufficient?